lib/ood_core/job/adapters/slurm.rb

```diff
@@ -178,6 +179,21 @@ def get_jobs(id: "", owner: nil, attrs: nil)
             return [{ id: id, state: 'undetermined' }]
           end
 
+          def accounts
+            user = Etc.getlogin
+            args = ['-nP', 'show', 'users', 'withassoc', 'format=account,user', 'where', "user=#{user}"]
+            upcase = ENV['OOD_UPCASE_ACCOUNTS'].nil? ? false : true
+
+            [].tap do |accts|
+              call('sacctmgr', *args).each_line do |line|
+                acct, _ = line.split('|')
+                accts << acct unless acct.nil?
+              end
+            end.uniq.map do |acct|
+              upcase ? acct.upcase : acct
```

Comment:
```suggestion
              :upcase ? acct.upcase : acct
```
I think this needs to be a symbol, wouldn't work for me locally without this.

Reply:
That's odd... it should be a boolean.

Reply:
I don't follow. Did you get an error or did the tests just fail?

```
irb(main):007:0> f = false
=> false
irb(main):008:0> :f ? true : false
=> true
irb(main):009:0> f ? true : false
=> false
```

Reply:
I just mocked the array of strings and ran it through using the pattern you had but noticed I got a failure without the symbol is all. I guess I would always think in Ruby to do this you'd need to use that symbol pattern:
```
['one', 'two', 'THREE'].map do |str|
  :upcase ? str.upcase : str
end
```
Was what I did to mock, but I may be misunderstanding this code.

Reply:
Ok, I see now there's an `upcase` variable doing something with a ternary above this. I need to take this apart more to understand all that's going on.